src/reducers/navReducer.js

```diff
@@ -0,0 +1,14 @@
+import { SET_ACTIVE_MENU } from '../actions/actionTypes';
+
+export default (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+    case SET_ACTIVE_MENU:
+      return { ...state, activeMenu: action.payload };
+    default:
+      return state;
+  }
+};
+
+const INITIAL_STATE = {
+  activeMenu: null
```

Comment:
Lets make this false instead of null - it will distinguish that this should be a bool value